syscall/unistd.c

```diff
@@ -145,6 +145,13 @@ int oe_nanosleep(struct oe_timespec* req, struct oe_timespec* rem)
     return ret;
 }
 
+int oe_clock_nanosleep(int clockid, int flag, struct oe_timespec* req, struct oe_timespec* rem)
```

Comment:
Does musl has `clock_nanosleep()` API? If so, we could put the test here: https://github.com/openenclave/openenclave/blob/master/tests/stdc/enc/enc.cpp#L172. Otherwise, we need to have something under tests/syscall/

Reply:
clock_nanosleep exists in musl. Added test here.